demos/python_demos/colorization/colorization_demo.py

```diff
@@ -0,0 +1,106 @@
+from openvino.inference_engine import IENetwork, IECore
+import cv2 as cv
+import numpy as np
+import os
+from argparse import ArgumentParser, SUPPRESS
+import xml.etree.ElementTree as ET
+import re
+import logging as log
+import glob
+import sys
+
+
+def build_arg():
+    parser = ArgumentParser(add_help=False)
+    in_args = parser.add_argument_group('Options')
+    in_args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Help with the script.')
+    in_args.add_argument("-m", "--model", help="Required. Path to .xml file with pre-trained model",
+                         required=True, type=str)
+    in_args.add_argument("-d", "--device",
+                         help="Optional. Specify target device for infer: CPU, GPU, FPGA, HDDL or MYRIAD. "
+                         "Default: CPU",
+                         default="CPU", type=str)
+    in_args.add_argument("-i", "--input", help="Required. Path to a folder with image files", required=True, type=str)
+    return parser
+
+
+def get_parameters(xml_path):
+    root = ET.parse(xml_path).getroot()
+    pattern = r'data\[([0-9]*\.*[0-9]*)\]'
+    scale_data = root.findall("./meta_data/cli_parameters/scale_values")[0]
+    mean_data = root.findall("./meta_data/cli_parameters/mean_values")[0]
+    scale_const = float(re.match(pattern, scale_data.attrib['value']).group(1))
+    mean_const = float(re.match(pattern, mean_data.attrib['value']).group(1))
+    return scale_const, mean_const
```

Comment:
Am I getting this right that scale and mean are stored in the source caffemodel?

Reply:
yes

Reply:
By the way, it could be a good idea to leave inline comment about it. Can you do it, please.

Reply:
I guess I got it wrong, scale and mean saved in prototxt + I wrote them into MO config

Reply:
The idea was to clarify that like `# scale and mean are stored in the source caffemodel` so every body would be clear about that.

Reply:
@Wovchena wrote